README.Rmd

```diff
@@ -24,22 +24,8 @@ knitr::opts_chunk$set(
 
 Antibody levels measured in a cross–sectional population sample can be translated into an estimate of the frequency with which seroconversions (infections) occur in the sampled population. In other words, the presence of many high antibody titers indicates that many individuals likely experienced infection recently and the burden of disease is high in the population, while low titers indicate a low frequency of infections in the sampled population and therefore a lower burden of disease.
 
-The **serocalculator** package was designed to use the longitudinal response characteristics using a set of modeled parameters characterizing the longitudinal response of the selected serum antibodies. 
+The **serocalculator** package was designed to use the longitudinal response characteristics using a set of modeled parameters characterizing the longitudinal response of the selected serum antibodies. More details on the underlying methods can be found in `vignette("serocalculator")` or the "Getting Started" link above.
```

Comment:
the "Getting Started" link won't appear on the repo version of this Readme, right? maybe hyperlink the URL (it will be at https://ucd-serg.github.io/serocalculator/articles/serocalculator.html)